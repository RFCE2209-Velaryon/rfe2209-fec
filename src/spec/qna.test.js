import renderer from 'react-test-renderer';
import React from 'react';

import QuestionsAndAnswers from '../components/Q&A/qna.jsx';

it('renders correctly', () => {
  const tree = renderer //tree makes snapshot (html comparison of component) - snapshot is what we expect
    .create(<QuestionsAndAnswers />)
    .toJSON();
    console.log(tree)
    expect(tree).toMatchSnapshot();
 });