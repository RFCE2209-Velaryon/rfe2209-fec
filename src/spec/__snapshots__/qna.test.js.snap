// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="main"
>
  <h1
    className="title"
  >
    Questions & Answers
  </h1>
  <form
    className="searchbar"
  >
    <input
      className="searchInput"
      onChange={[Function]}
      placeholder="HAVE A QUESTION? SEARCH FOR ANSWERS..."
      type="text"
    />
    <div>
      <i />
    </div>
  </form>
  <div
    className="questions-list"
  />
  <div
    className="footer"
  >
    <button
      className="buttons"
      onClick={[Function]}
    >
      ADD A QUESTION +
    </button>
  </div>
</div>
`;
